x-pack/plugins/beats/server/rest_api/tags/set.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import Joi from 'joi';
+import { get, values } from 'lodash';
+import { ConfigurationBlockTypes } from '../../../common/constants';
+import { CMServerLibs } from '../../lib/lib';
+import { wrapEsError } from '../../utils/error_wrappers';
+
+// TODO: add license check pre-hook
+// TODO: write to Kibana audit log file
+export const createSetTagRoute = (libs: CMServerLibs) => ({
+  config: {
+    validate: {
+      payload: Joi.object({
+        configuration_blocks: Joi.array().items(
+          Joi.object({
+            block_yml: Joi.string().required(),
+            type: Joi.string()
+              .only(values(ConfigurationBlockTypes))
```

Comment:
Not a huge deal but I noticed are using `_.values` here instead of `Object.values`. In general, per the Kibana JS style guide, we tend to prefer using native methods whenever possible, unless a utility library offers more expressiveness (https://github.com/elastic/kibana/blob/master/style_guides/js_style_guide.md#object--array-iterations-transformations-and-operations).

That said, this is somewhat subjective IMO so if you feel strongly about using `_.values` here, I think that's fine.

Reply:
Actually the TS config for kibana does not allow Object.values. At least for now loads was the best option